tests/testthat/test-list_datasets.R

```diff
@@ -0,0 +1,10 @@
+test_that("returns more than 0 datasets", {
+  # select the first row of the tibble and get the
+  # number of rows. If no datasets were returned
+  # this will be 0
+  expect_equal(nrow(slice(list_datasets(), 1)), 1)
```

Comment:
The test is failing because dplyr isn't loaded in the test environment.
```suggestion
  expect_equal(nrow(dplyr::slice(list_datasets(), 1)), 1)
```